topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -1752,15 +1769,14 @@ Galaxy is now configured with an admin user, a database, and a place to store da
 >    > Jun 28 17:07:58 gat-14.be.training.galaxyproject.eu systemd[1]: Failed to start Galaxy.
 >    > ```
 >    >
->    > Check your /srv/galaxy/config/galaxy.yml and ensure that it lines up exactly with what you expect. One possible cause here is the `uwsgi` subsection of `galaxy_config` was omitted and instead it used the default port and failed.
+>    > Check your /srv/galaxy/config/galaxy.yml and ensure that it lines up exactly with what you expect.
 >    {: .tip}
 >
 > 6. Some things to note:
 >
->    1. Later, after NGINX is set up and we're ready to connect to Galaxy, refreshing the page while Galaxy is restarting will wait (spin) until the process is ready rather than produce an error message, a nice feature of uWSGI
->    2. Although the playbook will restart Galaxy upon config changes, you will sometimes need to restart it by hand, which can be done with `systemctl restart galaxy`
->    3. You can use `journalctl -fu galaxy` to see the logs of Galaxy
->    4. You may have noticed after setting up Galaxy in the previous section, that Ansible printed the message "RESTARTER NOT IMPLEMENTED - Please restart Galaxy manually. You can define your own handler and enable it with `galaxy_restart_handler_name`." You should no longer see this message on future playbook runs, as enabling systemd has automatically set `galaxy_restart_handler_name` for us - the role now knows how to restart Galaxy!
+>    1. Although the playbook will restart Galaxy upon config changes, you will sometimes need to restart it by hand, which can be done with `systemctl restart galaxy`
```

Comment:
Do we need to use gravity here? Or are we going to wrap gravity within systemd? 

I like using Gravity - the way I do it is:
1. `sudo su - galaxy`
2. `source /srv/galaxy/venv/bin/activate`
3. `gravityctl restart` or `gravityctl graceful`

Reply:
I prefer to use systemd; it's a tool that people can know how to use before, can reuse after the training, can be used for other tools like NGINX.
In any case,  you can ever use gravityctl if you like but, in my opinion, the training should try to keep the access barrier low.

Reply:
Fair enough. :)

Reply:
yeah I think I'm fine keeping this systemctl/journalctl, especially in the context of: pulsar, slurm, etc all using it.